cookiecutter.json

```diff
@@ -1,5 +1,9 @@
 {
     "project_name": "sugardough",
+    "name": "Sugardough",
+    "project_description": "Add your project description",
     "project_db": "sugardough_db",
-    "project_author": "Mozilla Foundation"
+    "project_author": "Mozilla Foundation",
+    "license": "Apache 2.0",
```

Comment:
I wasn't sure on which license to use, so I looked on some Mozilla repos and saw that there was not a specific license used. I will change to ``MPL 2.0``.